Ignore/Inline Issue Tracker background Login Page.txt

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
In the interest of keeping the code clean, let's not commit anything that shouldn't be part of the project.